python/ray/train/v2/jax/config.py

```diff
@@ -20,14 +20,20 @@
 @dataclass
 class JaxConfig(BackendConfig):
     use_tpu: bool = False
+    use_gpu: bool = False
```

Comment:
Nit: this JaxConfig has a few redundant params with the ScalingConfig, and both of these are passed through from scaling config.

Plus, this is a public API that users can modify, so you could end up with ScalingConfig != JaxConfig which is a bit confusing.

Let's discuss and address this in a followup PR. Ok to merge for now